init.lua

```diff
@@ -71,6 +71,10 @@ minetest.register_node("castle_tapestries:tapestry", {
 	paramtype = "light",
 	paramtype2 = "colorwallmounted",
 	palette = "unifieddyes_palette_colorwallmounted.png",
+	collision_box = {
+		type = "fixed",
+		fixed = {0.4375,-0.5,-0.5,0.5,0.5,1.5},
```

Comment:
This limits the Z max axis to 1.5 instead of Y. Is that intentional?

Reply:
Yep intentional.
If you compare the values for the collision and selection box you will see that X also seems wrong
```
{0.4375,-0.5,-0.5,0.5,0.5,1.5}
{-0.5,-0.5,0.4375,0.5,1.5,0.5}
```
Dont ask me why but without that reordering you get a wrongly orientated collision box.